{"text": "2024-03-20 18:18:05 | INFO    | __main__:stash:131 | run - {'image': '20231121', 'version': '0.10.1.post2', 'role': 'EpicPlayer', 'headless': True}\n", "record": {"elapsed": {"repr": "0:00:01.279558", "seconds": 1.279558}, "exception": null, "extra": {"image": "20231121", "version": "0.10.1.post2", "role": "EpicPlayer", "headless": true}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/claim.py"}, "function": "stash", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 131, "message": "run", "module": "claim", "name": "__main__", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:18:05.763322+08:00", "timestamp": 1710929885.763322}}}
{"text": "2024-03-20 18:18:14 | INFO    | __main__:claim_epic_games:97 | Try to flush cookie - {'task': 'claim_epic_games'}\n", "record": {"elapsed": {"repr": "0:00:09.573795", "seconds": 9.573795}, "exception": null, "extra": {"task": "claim_epic_games"}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/claim.py"}, "function": "claim_epic_games", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 97, "message": "Try to flush cookie", "module": "claim", "name": "__main__", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:18:14.057559+08:00", "timestamp": 1710929894.057559}}}
{"text": "2024-03-20 18:18:17 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:00:12.613211", "seconds": 12.613211}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:18:17.096975+08:00", "timestamp": 1710929897.096975}}}
{"text": "2024-03-20 18:18:18 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xa2 in position 3: invalid start byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 607, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xa2 in position 3: invalid start byte\n", "record": {"elapsed": {"repr": "0:00:14.353113", "seconds": 14.353113}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xa2 in position 3: invalid start byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xa2 in position 3: invalid start byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:18:18.836877+08:00", "timestamp": 1710929898.836877}}}
{"text": "2024-03-20 18:18:21 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:00:16.579636", "seconds": 16.579636}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:18:21.063400+08:00", "timestamp": 1710929901.0634}}}
{"text": "2024-03-20 18:18:50 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:00:46.123258", "seconds": 46.123258}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:18:50.607022+08:00", "timestamp": 1710929930.607022}}}
{"text": "2024-03-20 18:18:50 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:00:46.123854", "seconds": 46.123854}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:18:50.607618+08:00", "timestamp": 1710929930.607618}}}
{"text": "2024-03-20 18:19:21 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 0}\n", "record": {"elapsed": {"repr": "0:01:17.205844", "seconds": 77.205844}, "exception": null, "extra": {"task": "authorize", "retry": 0}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:19:21.689608+08:00", "timestamp": 1710929961.689608}}}
{"text": "2024-03-20 18:19:23 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:01:19.292624", "seconds": 79.292624}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:19:23.776388+08:00", "timestamp": 1710929963.776388}}}
{"text": "2024-03-20 18:19:25 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xf2 in position 0: invalid continuation byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 607, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xf2 in position 0: invalid continuation byte\n", "record": {"elapsed": {"repr": "0:01:20.862008", "seconds": 80.862008}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xf2 in position 0: invalid continuation byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xf2 in position 0: invalid continuation byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:19:25.345772+08:00", "timestamp": 1710929965.345772}}}
{"text": "2024-03-20 18:19:27 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:01:23.389849", "seconds": 83.389849}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:19:27.873613+08:00", "timestamp": 1710929967.873613}}}
{"text": "2024-03-20 18:19:57 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:01:52.932977", "seconds": 112.932977}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:19:57.416741+08:00", "timestamp": 1710929997.416741}}}
{"text": "2024-03-20 18:19:57 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:01:52.933693", "seconds": 112.933693}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:19:57.417457+08:00", "timestamp": 1710929997.417457}}}
{"text": "2024-03-20 18:20:28 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 1}\n", "record": {"elapsed": {"repr": "0:02:24.022997", "seconds": 144.022997}, "exception": null, "extra": {"task": "authorize", "retry": 1}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:20:28.506761+08:00", "timestamp": 1710930028.506761}}}
{"text": "2024-03-20 18:20:30 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games'}\n", "record": {"elapsed": {"repr": "0:02:26.246239", "seconds": 146.246239}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "‚ÑπÔ∏è", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:20:30.730003+08:00", "timestamp": 1710930030.730003}}}
{"text": "2024-03-20 18:20:32 | ERROR    | hcaptcha_challenger.agents.playwright.control:handler:153 | 'utf-8' codec can't decode byte 0xce in position 3: invalid continuation byte - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py\", line 145, in handler\n    data = await response.json()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/async_api/_generated.py\", line 614, in json\n    return mapping.from_maybe_impl(await self._impl_obj.json())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 610, in json\n    return json.loads(await self.text())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/playwright/_impl/_network.py\", line 607, in text\n    return content.decode()\nUnicodeDecodeError: 'utf-8' codec can't decode byte 0xce in position 3: invalid continuation byte\n", "record": {"elapsed": {"repr": "0:02:28.102094", "seconds": 148.102094}, "exception": {"type": "UnicodeDecodeError", "value": "'utf-8' codec can't decode byte 0xce in position 3: invalid continuation byte", "traceback": true}, "extra": {}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "handler", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 153, "message": "'utf-8' codec can't decode byte 0xce in position 3: invalid continuation byte", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:20:32.585858+08:00", "timestamp": 1710930032.585858}}}
{"text": "2024-03-20 18:20:34 | DEBUG    | epic_games.agent:insert_challenge:215 | Attack challenge - {'stage': 'email_exists_prod'}\n", "record": {"elapsed": {"repr": "0:02:30.339317", "seconds": 150.339317}, "exception": null, "extra": {"stage": "email_exists_prod"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 215, "message": "Attack challenge", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:20:34.823081+08:00", "timestamp": 1710930034.823081}}}
{"text": "2024-03-20 18:21:04 | ERROR    | hcaptcha_challenger.agents.playwright.control:execute:615 | task interrupt <backcall> - {'reason': 'Failed to obtain challenge task'}\n", "record": {"elapsed": {"repr": "0:02:59.880959", "seconds": 179.880959}, "exception": null, "extra": {"reason": "Failed to obtain challenge task"}, "file": {"name": "control.py", "path": "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/hcaptcha_challenger/agents/playwright/control.py"}, "function": "execute", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 615, "message": "task interrupt <backcall>", "module": "control", "name": "hcaptcha_challenger.agents.playwright.control", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:21:04.364723+08:00", "timestamp": 1710930064.364723}}}
{"text": "2024-03-20 18:21:04 | DEBUG    | epic_games.agent:insert_challenge:218 | Parse result - {'stage': 'email_exists_prod', 'result': <Status.CHALLENGE_BACKCALL: 'backcall'>}\n", "record": {"elapsed": {"repr": "0:02:59.881699", "seconds": 179.881699}, "exception": null, "extra": {"stage": "email_exists_prod", "result": "backcall"}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "insert_challenge", "level": {"icon": "üêû", "name": "DEBUG", "no": 10}, "line": 218, "message": "Parse result", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:21:04.365463+08:00", "timestamp": 1710930064.365463}}}
{"text": "2024-03-20 18:21:35 | WARNING    | epic_games.agent:authorize:266 | ÊâßË°åË∂ÖÊó∂ - {'task': 'authorize', 'retry': 2}\n", "record": {"elapsed": {"repr": "0:03:30.981592", "seconds": 210.981592}, "exception": null, "extra": {"task": "authorize", "retry": 2}, "file": {"name": "agent.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "‚ö†Ô∏è", "name": "WARNING", "no": 30}, "line": 266, "message": "ÊâßË°åË∂ÖÊó∂", "module": "agent", "name": "epic_games.agent", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:21:35.465356+08:00", "timestamp": 1710930095.465356}}}
{"text": "2024-03-20 18:21:35 | ERROR    | __main__:run:158 | An error has been caught in function 'run', process 'MainProcess' (3772), thread 'MainThread' (125128776235904): - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/claim.py\", line 158, in run\n    await agent.stash()\n  File \"/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/claim.py\", line 151, in stash\n    await self.claim_epic_games(context)\n  File \"/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/claim.py\", line 98, in claim_epic_games\n    if await epic.authorize(page):\n  File \"/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/epic_games/agent.py\", line 269, in authorize\n    raise RuntimeError(f\"Failed to flush token - agent={self.__class__.__name__}\")\nRuntimeError: Failed to flush token - agent=EpicGames\n", "record": {"elapsed": {"repr": "0:03:31.149165", "seconds": 211.149165}, "exception": {"type": "RuntimeError", "value": "Failed to flush token - agent=EpicGames", "traceback": true}, "extra": {}, "file": {"name": "claim.py", "path": "/home/runner/work/epic-awesome-gamer/epic-awesome-gamer/epic/src/claim.py"}, "function": "run", "level": {"icon": "‚ùå", "name": "ERROR", "no": 40}, "line": 158, "message": "An error has been caught in function 'run', process 'MainProcess' (3772), thread 'MainThread' (125128776235904):", "module": "claim", "name": "__main__", "process": {"id": 3772, "name": "MainProcess"}, "thread": {"id": 125128776235904, "name": "MainThread"}, "time": {"repr": "2024-03-20 18:21:35.632929+08:00", "timestamp": 1710930095.632929}}}
